src/test/java/edu/kis/powp/jobs2d/command/ImmutableCommandBuilderTest.java

```diff
@@ -0,0 +1,10 @@
+package edu.kis.powp.jobs2d.command;
+
+public class ImmutableCommandBuilderTest {
+
+    public static void main(String[] args) {
+        ImmutableCommandBuilder ImmutableCommandBuilder = new ImmutableCommandBuilder(5).setName("Name").addCommandsList("CommandList").build();
```

Comment:
There should be also a test "clickable" from the application (e.g. returning specific shape command)